src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -432,6 +434,22 @@ def get_role(self, name):
                 raise RoleDoesNotExist(name)
             return self.roles[name]
 
+    def set_attempt(self, username, success=False):
```

Comment:
I now see the comment by @s0nea... This is a kind of swiss-army knife method, it both increments and resets a value and I think that's not a good practice. `increment_attempt` should +1 to attempts, and `reset_attemp` should zero that. That's 2 different methods.

Reply:
Okay, I'll create 2 methods - one for incrementing and other for resetting. @epuertat

Reply:
Thanks!

Reply:
Done, thanks.